packages/dx-react-grid/docs/reference/virtual-table.md

```diff
@@ -38,6 +38,15 @@ stubRowComponent | ComponentType&lt;[Table.RowProps](#tablerowprops)&gt; | | A c
 stubCellComponent | ComponentType&lt;[Table.CellProps](table.md#tablecellprops)&gt; | | A component that renders a stub table cell if the cell value is not provided.
 stubHeaderCellComponent | ComponentType&lt;[Table.CellProps](table.md#tablecellprops)&gt; | | A component that renders a stub header cell if the cell value is not provided.
 messages? | [Table.LocalizationMessages](table.md#localization-messages) | | An object that specifies the localization messages.
+onRowIndexChange? | (index: number) => void | | Handles top row index changes.
```

Comment:
We should consider combining these events into one with args = { index, rowId } because their purpose is the same